packages/paste-website/src/pages/getting-started/design/index.mdx

```diff
@@ -0,0 +1,400 @@
+your design work.
+
+### Abstract
+
+All product designers across Twilio use <Anchor href="https://www.abstract.com/">Abstract</Anchor> to host and version control
+their design files. We also use Abstract to link Paste Sketch libraries and
+manage library updates (more on that later). Connect with your design manager
+to get added to the Twilio team in Abstract.
+
+<Callout variant="information">
+  <CalloutTitle>
+    Getting familiar with Abstract
+  </CalloutTitle>
+  <CalloutText>
+    If you are not yet familiar with Abstract, please take some time to learn
+    about <Anchor href="https://www.abstract.com/how-it-works">how it works</Anchor>,
+    and read through these resources
+    on <Anchor href="https://help.abstract.com/best-practices/">best practices</Anchor>.
+  </CalloutText>
+</Callout>
+
+### Sketch
+
+Product designers across Twilio use Sketch to design their UIs. You
+can <Anchor href="https://twilio.service-now.com/sp/">file a ticket with IT</Anchor> to
+get a Sketch license. In the meantime, you
+can <Anchor href="https://www.sketch.com/try/">download the free trial</Anchor>
+on your work laptop to get you up and running.
+
+### Font files
+
+#### Inter
+
+If you are using the Default theme in Paste, you will need
+to <Anchor href="https://github.com/rsms/inter/releases/download/v3.15/Inter-3.15.zip">download Inter.</Anchor>
+It’s a free font. After you download the files, follow the instructions in the
+`install-mac.txt` file in your download. We recommend following the steps using Font Book.
+
+#### Fira Mono
+
+If you are using any theme in Paste, you will need
+to <Anchor href="https://fonts.google.com/specimen/Fira+Mono?query=fira+mono">download Fira Mono</Anchor>,
+our monospace typeface for code. It’s a free Google font.
+
+#### SF Pro
+If you are using any theme in Paste, you will need to <Anchor href="https://developer.apple.com/fonts/">download SF Pro</Anchor>, the
+system font for Apple platforms. We represent native web element styles (like option lists in <Anchor href="/form-elements/select/">Select</Anchor>) in
+our design assets as macOS styles.
+
+#### Whitney
+
+If you are using the Console theme in Paste, you will need to download Whitney.
+Reach out to us in `#help-design-system` if you need access to the Whitney font
+files for your work.
+
+## How to use Sketch libraries
+
+For every Abstract project that you work on where you plan on using Paste, you
+will need to link the necessary Paste Sketch libraries in order to use the
+Sketch symbols, text styles, and layer styles.
+
+<Callout variant="information">
+  <CalloutTitle>
+    Tips on linking libraries in Abstract
+  </CalloutTitle>
+  <CalloutText>
+    If you haven’t linked libraries in Abstract before,
+    follow <Anchor href="https://help.abstract.com/libraries/link-libraries/">this guide</Anchor> to learn how.
+  </CalloutText>
+</Callout>
+
+### Paste design tokens
+
+<Flex>
```

Comment:
This is interesting, and might be a good _systemized_ way to display status across the system. However, the tooltip text here gives me more info than the text next to it. Also, without any borders, I skipped it on my first read through the PR.

Reply:
@aayushpi I added borders around these in my most recent commit - lmk what you think